Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -2279,6 +2284,16 @@ int CLuaPedDefs::SetPedAnimationProgress(lua_State* luaVM)
     return 1;
 }
 
+float CLuaPedDefs::GetPedAnimationProgress(CClientEntity* ped)
+{
+    return CStaticFunctionDefinitions::GetPedAnimationProgress(*ped);
```

Comment:
CStaticFunctionDefinitions isn't needed for the new code.

Reply:
> CStaticFunctionDefinitions isn't needed for the new code.

You mean to instead directly put the code in the luadef?

Reply:
Yes, your logic is short enough to be placed directly in the luadef.

Reply:
It would clutter the luadef with `game` headers but alright

Reply:
It would clutter either `LuaDefs` or `CStaticFunctionDefinitions`. But `CStaticFunctionDefinitions` is far more cluttered. On the other hand `LuaDefs*` is devided by groups which brings more order. `CStaticFunctionDefinitions` is outdated and have to be deleted in the future.